mariadb/meta/main.yml

```diff
@@ -4,7 +4,7 @@ dependencies: []
 
 galaxy_info:
   author: 'Adfinis SyGroup AG'
-  description: 'Install and manage mariadb/mysql server'
+  description: 'Install and manage mariadb/mysql server. pyOpenSSL is locally also required.'
```

Comment:
Maybe write `pyOpenSSL is required on local and remote host.`?

Reply:
But on the remote host, it will be installed if not present, so technically it isn't a requirement.

Reply:
ACK